egs/wsj/s5/steps/dict/apply_g2p_phonetisaurus.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 2014  Johns Hopkins University (Author: Yenda Trmal)
+# Copyright 2016  Xiaohui Zhang
+#           2018  Ruizhe Huang
+# Apache 2.0
+
+# This script applies a trained Phonetisarus G2P model to
+# synthesize pronunciations for missing words (i.e., words in
+# transcripts but not the lexicon), and output the expanded lexicon.
```

Comment:
explain here that, the user could specify either nbest or pmass option to determine the number of output pronunciation variants, or use them together.